src/shared/bg_parse.cpp

```diff
@@ -803,7 +803,7 @@ void BG_ParseBuildableAttributeFile( const char *filename, buildableAttributes_t
 	else if ( !( defined & BUILDTIME) ) { token = "buildTime"; }
 	else if ( !( defined & NORMAL) ) { token = "minNormal"; }
 
-	if ( !Util::is_empty(token) )
+	if ( strlen( token ) > 0 )
```

Comment:
why doing the change and reverting it? what do you mean by "till daemon is updated" ?

Reply:
I guess the idea was to define an `is_empty` function in Daemon.

We could merge using the squash option to avoid putting this into the history. Sounds good?

Reply:
Squash is good.
Now replaced with `if ( *token )`, which doesn't require anything new to be added.

Out of curiosity, when will we update daemon to the upstream version?

Reply:
Squash is good.
Now replaced with if ( *token ), which doesn't require anything new to be added.

Out of curiosity, when will we update daemon to the upstream version?

Reply:
Oh, i see, @illwieckz already updated daemon. Thank you!

Reply:
At least, I would move commits in a way they follow immediately (maybe use “revert” word instead of “restore”). Like commit 1, commit 2, commit revert. One day we can revert the revert the day the feature is available. With squash we would lose that.

Reply:
Note that if you want to implement `is_empty` in engine and thinks it's worth of it, don't hesitate to do it, then it's fine if you update the submodule yourself in the pr.

Reply:
Removed affected commits, now history is clean. Now `is_empty` looks too local 4 me to add to daemon. 

But thanks for suggestions!